cmd/framesd/framesd.go

```diff
@@ -88,7 +88,7 @@ func main() {
 		}
 	}
 
-	hsrv, err := framesHttp.NewServer(cfg, config.httpAddr, framesLogger, historyServer)
+	hsrv, err := framesHttp.NewServer(cfg, config.httpAddr, framesLogger, historyServer, Version)
```

Comment:
It doesn't look like `Version` is being set

Reply:
it does, by the build process
see in makefile -> FRAMES_BUILD_COMMAND